tasks/build.yml

```diff
@@ -4,35 +4,39 @@
   become: yes
   apt_repository:
     repo: ppa:ondrej/php
-  when: ((php.version == "php5.5") or (php.version == "php5.6") or (php.version == "php7.0"))
 
 - name: Install PHP
   become: yes
   apt:
     update_cache: yes
     name: "{{ item }}"
   with_items:
-    - "{{ php.version }}"
-    - "{{ php.version }}-common"
-    - "{{ php.version }}-fpm"
-    - "{{ php.version }}-cli"
+    - "{{ sansible_php_version }}"
+    - "{{ sansible_php_version }}-common"
+    - "{{ sansible_php_version }}-fpm"
+    - "{{ sansible_php_version }}-cli"
+
+- name: Don't Install mcrypt Module if PHP Version is 7.2
+  set_fact:
+    sansible_php_modules: "{{ sansible_php_modules | difference(['php7.2-mcrypt']) }}"
+  when: sansible_php_version == "php7.2"
```

Comment:
I don't think we will need this anymore since we are removing mcrypt as a default.